docs/getting-started/configure.mdx

```diff
@@ -0,0 +1,77 @@
+# Configuration
+
+## Customize how use-cases are previewed
+
+Widgetbook allows developers to fully customize how use-cases are previewed based on the selected add-on properties.
```

Comment:
Two headlines look a bit odd. Maybe add some text below Configuration?

Reply:
I think for now, it's good to keep Configure, later I can split it to more specific names.